src/main/webapp/style.css

```diff
@@ -0,0 +1,90 @@
+body{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.navbar{
+  width: 100%;
+  max-width: 1400px; 
+  flex-flow: row nowrap;
+}
+
+.form-line{
+    max-width: 60vw;
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    flex-grow: 1;
+}
+
+.wrapper{
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  background-color: #A9A9A9;
+  width: 100%;
+}
+
+#header{
+  height: 8vh;
+  min-height: 50px;
+}
+
+#footer{
+  position: fixed;
```

Comment:
Not sure if the footer needs to be fixed, it doesn't contain information that is useful always - everywhere. Instead, the header would make more sense being fixed positioned.

Reply:
I guess it looks nicer? We can change it later when we discuss the frontend as it's only css